### PR TITLE
Node consistency 71 80

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose
The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**71) NODE: Math.DivRem**
-CHANGES:
Output name: int
Output int description: The remainder of the division

**72) NODE: Math.Cosh**
-CHANGES:
Output double description:  Hyperbolic cosine of the angle

**73) NODE: Math.Cos**
Output name: double

**74) NODE: Math.Atan2**
-CHANGES:
Output name: double

**75) NODE: Math.Atan**
-CHANGES:
Output name: double

**76) NODE: Math.Asin**
Output name: double

**77) NODE: Math.Acos**
-CHANGES:
Output name: double

**78) NODE: List.SortByKey**
-CHANGES:
Output: sortedList,sortedKeys
-Input Description:
keys: list of keys, keys have to be sortable (e. g. numbers, strings)
Output Description: 
sortedList:  type: var[]..[]
sortedKeys:  type: var[]..[]

**79) NODE: List.SortByFunction**
-CHANGES:
Input name: sortFunction
Input sortFunction description: function used to sort list.
Output sortedList description: type: var[]..[]

**80) NODE: List.Sublists**
-CHANGES:
Output lists description: type: var[]..[]

image

### Reviewers
@SHKnudsen
@StudioLE
@MarkThorley
@anrova

### FYIs
-No FYI

### CODE REVIEW

File: src/Libraries/CoreNodes/List.cs [hunk 1]
```diff
@@ -441,9 +441,9 @@ public static IDictionary Deconstruct(IList list)
         ///     Sort list based on its keys
         /// </summary>
         /// <param name="list">list to be sorted</param>
-        /// <param name="keys">list of keys</param>
-        /// <returns name="sorted list">sorted list</returns>
-        /// <returns name="sorted keys">sorted keys</returns>
+        /// <param name="keys">list of keys, keys have to be sortable (ex: numbers,strings)   </param>
+        /// <returns name="sortedList">List sorted using keys</returns>
+        /// <returns name="sortedKeys">Keys sorted</returns>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/List.cs]: To me this seems unnecessary, `Sorted list` and `Sorted keys` is very clear.

File: src/Libraries/CoreNodes/Math.cs [hunk 2]
```diff
@@ -231,7 +231,7 @@ public static long Abs(long integer)
         ///     Finds the inverse cosine, the angle whose cosine is the given ratio.
         /// </summary>
         /// <param name="ratio">The cosine of the angle, a number in the range [-1, 1].</param>
-        /// <returns name="angle">The angle whose cosine is the input ratio.</returns>
+        /// <returns name="double">The angle whose cosine is the input ratio. (double)</returns>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/Math.cs]: `(double)` seems unnecessary. It already says double in the name.

File: src/Libraries/CoreNodes/Math.cs [hunk 3]
```diff
@@ -242,7 +242,7 @@ public static double Acos(double ratio)
         ///     Finds the inverse sine, the angle whose sine is the given ratio.
         /// </summary>
         /// <param name="ratio">The sine of the angle, a number in the range [-1, 1].</param>
-        /// <returns name="angle">The angle whose sine is the input ratio.</returns>
+        /// <returns name="double">The angle whose sine is the input ratio (double)</returns>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/Math.cs]: `(double)`

File: src/Libraries/CoreNodes/Math.cs [hunk 4]
```diff
@@ -253,7 +253,7 @@ public static double Asin(double ratio)
         ///     Finds the inverse tangent, the angle whose tangent is the given ratio.
         /// </summary>
         /// <param name="ratio">The tangent of the angle.</param>
-        /// <returns name="angle">The angle whose tangent is the input ratio.</returns>
+        /// <returns name="double">The angle whose tangent is the input ratio (double)</returns>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/Math.cs]: `(double)`

Comment (SHKnudsen) [reply src/Libraries/CoreNodes/Math.cs]: this goes for all of the following `(double)`

File: src/Libraries/CoreNodes/Math.cs [hunk 5]
```diff
@@ -312,7 +312,7 @@ public static double Cosh(double angle)
         /// </summary>
         /// <param name="dividend">The number to be divided.</param>
         /// <param name="divisor">The number to be divided by.</param>
-        /// <returns name="remainder">The remainder of the division.</returns>
+        /// <returns name="integer">The remainder of the division (integer)</returns>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/Math.cs]: `(integer)`

File: src/Libraries/CoreNodes/List.cs (hunk 1 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/List.cs]: This is to keep consistency with the rest of the outputs (camelCase)
Maybe the description is a bit obvious.

File: src/Libraries/CoreNodes/Math.cs (hunk 2 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/Math.cs]: adressed in 
0cb98d5

File: src/Libraries/CoreNodes/Math.cs (hunk 4 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/Math.cs]: adressed in 
0cb98d5

File: src/Libraries/CoreNodes/Math.cs (hunk 5 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/Math.cs]: adressed in 
0cb98d5

File: src/Libraries/CoreNodes/List.cs (hunk 1 above)
Comment (SHKnudsen) [reply src/Libraries/CoreNodes/List.cs]: Yea not talking about the names, that makes sense. Just think the new description are unnecessary.

File: src/Libraries/CoreNodes/Math.cs (hunk 3 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/Math.cs]: addressed in 0cb98d5

File: src/Libraries/CoreNodes/List.cs (hunk 1 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/List.cs]: addressed in 172ba45

**Pull request merged**